astro.config.mjs

```diff
@@ -4,6 +4,7 @@ import starlightBlog from 'starlight-blog';
 
 // https://astro.build/config
 export default defineConfig({
+	site: 'https://sarah-starlight.netlify.app',
```

Comment:
```suggestion
	site: 'https://www.rainsberger.ca',
```
This should probably be my domain, right?

Reply:
Oh, definitely, I picked the one from the GitHub repo description:

![image](https://github.com/sarah11918/starlight-blog/assets/494699/6aec7e6b-53e8-4051-ace4-2d604f2efacd)